analytic_engine/src/table/data.rs

```diff
@@ -315,7 +317,23 @@ impl TableData {
 
         let memtable_factory: MemTableFactoryRef = match opts.memtable_type {
             MemtableType::SkipList => Arc::new(SkiplistMemTableFactory),
-            MemtableType::Columnar => Arc::new(ColumnarMemTableFactory),
+            MemtableType::Column => Arc::new(ColumnarMemTableFactory),
+        };
+
+        // Wrap it by `LayeredMemtable`.
+        let mutable_segment_switch_threshold = opts
+            .layered_memtable_opts
+            .mutable_segment_switch_threshold
+            .0 as usize;
+        let enable_layered_memtable = mutable_segment_switch_threshold > 0;
+        let memtable_factory = if enable_layered_memtable {
+            debug!("Enable layered memtable, mutable_segment_type:{}, mutable_segment_switch_threshold:{mutable_segment_switch_threshold}", opts.memtable_type.to_string());
```

Comment:
Is this debugging log necessary? How about removing it? (At least avoid the `to_string` clone.)

Reply:
It seems useless actually, we can get it by `show create table xxx`.